### PR TITLE
[skip vbump] Closes #534 #532 #529 Release 1.4...Release the 🦑 !!!

### LINKED ISSUE #534: General Issue: Release admiraldev 1.4 🦑
### Background Information

Release admiraldev 1.4

### Definition of Done

Release the 🦑 !!!!!!!

### PR DESCRIPTION
**Pull request by bms63**
See [Release Notes](https://github.com/pharmaverse/admiral/discussions/2394)

Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/dev/CONTRIBUTING.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `main` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [ ] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#deprecation)?
- [ ] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [ ] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` under the header `# admiral (development version)` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [ ] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiraldev/reference/index.html)" page. 
- [ ] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [ ] Link the issue in the Development Section on the right hand side.
- [ ] Address all merge conflicts and resolve appropriately
- [ ] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

**Comment by bms63** [comment]
opinions on experimental function section?  I put verbatim in admiral reference page

**Comment by bms63** [comment]
Also where does this stuff exist? 
image

Can someone rerun the manual pages - somethign is amiss on my side.

**Comment by bms63** [comment]
> Also where does this stuff exist? image
> 
> Can someone rerun the manual pages - somethign is amiss on my side.

ah I see it in inst, when I run this is return a blank

File: R/compat_friendly_type.R [hunk 1]
```diff
@@ -14,146 +19,15 @@
...
-
-# Used in building `friendly_type_of()` above
-.rlang_stop_unexpected_typeof <- function(x, call = rlang::caller_env()) {
-  rlang::abort(
-    sprintf("Unexpected type <%s>.", typeof(x)),
-    call = call
-  )
 }
 
 #' @param x The object type which does not conform to `what`. Its
```
[the remark this answers is not in the document]
Comment (bms63) [reply R/compat_friendly_type.R]: just move to error msg or ask politely if being used with deprecate_inform() ??
